src/daft-scan/src/glob.rs

```diff
@@ -106,13 +106,19 @@ impl GlobScanOperator {
                         delimiter,
                         has_headers,
                         double_quote,
+                        quote,
+                        escape_char,
+                        comment,
                         ..
                     }) => {
                         let (schema, _, _, _, _) = daft_csv::metadata::read_csv_schema(
                             first_filepath,
                             *has_headers,
                             Some(delimiter.as_bytes()[0]),
                             *double_quote,
+                            Some(*quote as u8),
+                            Some(*escape_char as u8),
+                            Some(*comment as u8),
```

Comment:
Same here in terms of fallible casting + keeping `escape_char` and `comment` as `Option`s:
```suggestion
                            Some(u8::try_from(quote)?),
                            escape_char.map(|c| u8::try_from(c)).transpose()?,
                            comment.map(|c| u8::try_from(c)).transpose()?,
```